geometric_kernels/basis.py

```diff
@@ -0,0 +1,219 @@
+"""
```

Comment:
This file seems like largely copy-pasted from https://github.com/GPflow/GeometricKernels/blob/main/geometric_kernels/spaces/eigenfunctions.py. Not sure why that is necessary?